src/fake_schema.ts

```diff
@@ -25,14 +25,14 @@ type FakeArgs = {
 type ExamplesArgs = {
   values: [any];
 };
-type ListLenghtArgs = {
+type listLengthArgs = {
```

Comment:
@knidarkness Looks great 👍 
One issue through can you please keep an upper case letter at the start of the type name.

Reply:
@IvanGoncharov ouch, my fault on that. Reverted to initial capital-first naming.